javascript/javascript3/week3/script.js

```diff
@@ -0,0 +1,285 @@
+//use class way original data
+let c = document.getElementById("myCanvas");
+c.width = window.innerWidth;
+c.height = window.innerHeight;
+let ctx = c.getContext("2d");
+class Circle {
+    constructor(x, y, r, startAngle, endAngle, fillColor) {
+        this.x = x,
+            this.y = y,
+            this.r = r,
+            this.startAngle = startAngle,
+            this.endAngle = endAngle,
+            this.fillColor = fillColor
+    }
+    draw() {
+
+        ctx.beginPath();
+        ctx.arc(
+            this.x,
+            this.y,
+            this.r,
+            this.startAngle,
+            this.endAngle,
+            this.fillColor);
+        ctx.stroke();
+        ctx.fillStyle = this.fillColor;
+        ctx.fill();
+    }
+}
+const timeInterval = setInterval(() => {
+    let allCircles = [];
+    let amount = 2;
+    let createCircle = function (circle) {
```

Comment:
const

Reply:
inside the function, should we still use const?

Reply:
Yeah, when you're not changing the value.